[2025-03-16T03:52:56.001+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T03:52:56.015+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection manual__2025-03-16T03:48:17.725362+00:00 [queued]>
[2025-03-16T03:52:56.022+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection manual__2025-03-16T03:48:17.725362+00:00 [queued]>
[2025-03-16T03:52:56.022+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 6
[2025-03-16T03:52:56.035+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.check_minio_connection> on 2025-03-16 03:48:17.725362+00:00
[2025-03-16T03:52:56.042+0000] {standard_task_runner.py:72} INFO - Started process 572 to run task
[2025-03-16T03:52:56.045+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.check_minio_connection', 'manual__2025-03-16T03:48:17.725362+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpfaubdg9x']
[2025-03-16T03:52:56.046+0000] {standard_task_runner.py:105} INFO - Job 120: Subtask bronze_layer_group.check_minio_connection
[2025-03-16T03:52:56.091+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.check_minio_connection manual__2025-03-16T03:48:17.725362+00:00 [running]> on host c891f3a5df31
[2025-03-16T03:52:56.162+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.check_minio_connection' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T03:48:17.725362+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T03:48:17.725362+00:00'
[2025-03-16T03:52:56.163+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T03:52:56.166+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T03:52:56.168+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T03:52:56.169+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T03:52:56.169+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T03:52:56.694+0000] {python.py:240} INFO - Done. Returned value was: True
[2025-03-16T03:52:56.712+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T03:52:56.712+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.check_minio_connection, run_id=manual__2025-03-16T03:48:17.725362+00:00, execution_date=20250316T034817, start_date=20250316T035256, end_date=20250316T035256
[2025-03-16T03:52:56.741+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T03:52:56.750+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
